extension/content.js

```diff
@@ -344,6 +344,18 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="refined-github-project-new-link">Add a project</a>`;
+	$('#projects-feature').next('p.note').after(projectNewLink);
```

Comment:
The ideal method for selecting would [`$.fn.nextAll()`](https://api.jquery.com/nextAll/):

```js
$('#projects-feature').nextAll('p.note')
```

I've tested it on the Settings page and it finds the needed element. We could also use a CSS selector:
```js
$('#projects-feature ~ p.note')
```